sec_certs/sample/common_criteria.py

```diff
@@ -708,13 +708,18 @@ def _get_dependency_cves(self, dset: "CCDataset", dependency_type: DependencyCVE
             return None
 
         vulnerabilities = set()
+        dataset_cert_id_occurrences = dset.get_dataset_cert_ids_occurrences()
 
         for cert_id in references:
+            cert_id_occurrence = dataset_cert_id_occurrences.get(cert_id, None)
+
+            # Not in dataset or more than 2 occurrences in dataset -> skip
+            if cert_id_occurrence is None or cert_id_occurrence >= 2:
+                continue
```

Comment:
Suggestion: Can we count the total number of thrown-away certificates and `log.warning()` this number?